api/store/mongo/migrations.go

```diff
@@ -141,6 +143,31 @@ var migrations = []migrate.Migration{
 			return err
 		},
 	},
+	{
+		Version: 9,
+		Up: func(db *mongo.Database) error {
+			cursor, err := db.Collection("devices").Find(context.TODO(), bson.D{})
+			if err != nil {
+				return err
+			}
+			defer cursor.Close(context.TODO())
+			for cursor.Next(context.TODO()) {
+				device := new(models.Device)
+				err := cursor.Decode(&device)
+				if err != nil {
+					return err
+				}
+
+				device.Name = strings.ToLower(device.Name)
+				_, err = db.Collection("devices").UpdateOne(context.TODO(), bson.M{"uid": device.UID}, bson.M{"$set": bson.M{"name": strings.ToLower(device.Name)}})
```

Comment:
If it fails here, we need to do something. We cannot blindly ignore the error.

Reply:
now i'm returning thr error everytime the update fails. It's ok?